es/extend_your_application/README.md

```diff
@@ -48,7 +48,8 @@ Vamos a crear una URL en `urls.py` para nuestro *view* `post_detail`!
 
 Queremos crear una URL que apunte a Django a una *view* denominada `post_detail`, que mostrará una entrada del blog. Agrega la línea `url (r'^post/(?P<pk>[0-9]+)/$', views.post_detail),` al archivo `blog/urls.py`. Debería tener este aspecto:
 
-``` from django.conf.urls import include, url
+``` python
```

Comment:
If you could please keep `django` as the syntax highlighting for django code it would be awesome. Thank you! :))